setup.py

```diff
@@ -13,9 +14,10 @@
 
 requirements = ['koji', 'toolchest']
 
-setup_requirements = ['pytest-runner', ]
+needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
```

Comment:
This should not be needed.

Reply:
not absolutely needed correct but allows setup.py to not have to pull pytest-runner if not doing any testing related stuff

Reply:
I see that, not sure I like the pattern however. Adding logic blocks just to control if it's needed or not seems like a code smell to me. Maybe we don't use pytest-runner at all.

Reply:
After doing some more research, I guess this is a common pattern. Still not a fan, but better then writing our own test runner.

Reply:
I see the example, not sure I love it either, but let's go with it for now, and if we later find we don't need it, we can drop this bit?